### PR TITLE
Fix/959 non tracing processes

### PR DESCRIPTION
**Pull request by seanbrookes**
connected to #1565
connected to strongloop-internal/scrum-nodeops#937

### CODE REVIEW

File: client/www/scripts/modules/tracing/tracing.controllers.js [hunk 1]
```diff
@@ -552,11 +551,19 @@ Tracing.controller('TracingMainController', [
         $scope.resetTracingCtx();
         $scope.selectedPMHost = host;
         $scope.tracingCtx.currentManagerHost = $scope.selectedPMHost;
+        // establish current process count
```
Comment (sam-github) [inline client/www/scripts/modules/tracing/tracing.controllers.js]: Assuming that changePmHost is descriptive, this means that the current cluster size is derived by counting processes (fragile), and is done once, on first connection to the pm? So, if I do `slc ctl set-size` in the background after the change has occurred, tracing will permanently think the size is something that it is not?

@seanbrookes we should do an in-person review, please.

**Comment by anthonyettinger** [comment]
LGTM

File: client/www/scripts/modules/tracing/tracing.controllers.js (hunk 1 above)
Comment (seanbrookes) [inline client/www/scripts/modules/tracing/tracing.controllers.js]: good idea.  I'm finding the set-size to unreliable as well.

**Comment by sam-github** [comment]
Closing as duplicate of the already merged #1576

**Pull request closed without merging**